src/conic_algorithm.jl

```diff
@@ -2106,27 +2099,32 @@ function print_finish(m::PajaritoConicModel)
         else
             @printf "\nIterative algorithm summary:\n"
         end
-        @printf " - Total time (s)       = %14.2e\n" m.logs[:total]
         @printf " - Status               = %14s\n" m.status
-        @printf " - Best feasible obj.   = %+14.6e\n" m.best_obj
-        @printf " - OA obj. bound        = %+14.6e\n" m.mip_obj
+        @printf " - Best feasible        = %+14.6e\n" m.best_obj
+        @printf " - Best bound           = %+14.6e\n" m.mip_obj
         @printf " - Relative opt. gap    = %14.3e\n" m.gap_rel_opt
+        @printf " - Total time (s)       = %14.2e\n" m.logs[:total]
+    end
+
+    if m.gap_rel_opt < -10*m.rel_gap
+        warn("Best feasible value is smaller than best bound: this may indicate the final solution was constructed from a conic solver solution with significant infeasibilities\n")
```

Comment:
we do print warnings with some explanations when things go wrong (and suggestions for options to change), but we do that in the algorithm rather than at the end here